streamer/src/quic.rs

```diff
@@ -446,12 +451,14 @@ fn handle_connection(
     stake: u64,
 ) {
     tokio::spawn(async move {
-        debug!(
-            "quic new connection {} streams: {} connections: {}",
+        info!(
```

Comment:
don't think these should be `info` by default

Reply:
Okay -- I used these info to debug connection issues. I can turn them into debug level.